### ISSUE TITLE
rescuezilla Swedish Languge

### ISSUE THREAD

**Issue by AliensInc** [rescuezilla/rescuezilla#186]
Swedish language, not that I use it though but some might. I use English for everything all back since the 90's.
It's so much easier to find help in english :)
Hopefully no errors in there but we'll see.

[rescuezilla_Swedish Language.zip](https://github.com/rescuezilla/rescuezilla/files/5947149/rescuezilla_Swedish.Language.zip)

**Comment by shasheene** [comment]
Great work, @AliensInc!

The next version is Rescuezilla v2.2 scheduled for May 2021 which is too far away, so I have created a build that's simply Rescuezilla v2.1.3 plus your translation: [rescuezilla-2.1.3-4-ga461f3bd-64bit.groovy.iso](https://drive.google.com/u/0/uc?id=1wrjYb8ENoExElyrHFsuCaTAKjPHfibh5&export=download). Feel free to distribute it on forums or anywhere else, as you see fit.

<a href="https://user-images.githubusercontent.com/1414379/107301790-169a0f80-6acc-11eb-84fd-887c625728a4.png"><img width=350 src="https://user-images.githubusercontent.com/1414379/107301790-169a0f80-6acc-11eb-84fd-887c625728a4.png"></a> <a href="https://user-images.githubusercontent.com/1414379/107301799-18fc6980-6acc-11eb-8652-2f637cbcbd06.png"><img width=350 src="https://user-images.githubusercontent.com/1414379/107301799-18fc6980-6acc-11eb-8652-2f637cbcbd06.png"></a> <a href="https://user-images.githubusercontent.com/1414379/107301796-1863d300-6acc-11eb-969c-e6b3a6aa229b.png"><img width=780 src="https://user-images.githubusercontent.com/1414379/107301796-1863d300-6acc-11eb-969c-e6b3a6aa229b.png"></a>

For the keyboard I have chose the default Swedish keyboard layout and _not_ any of the following variants:
```
  nodeadkeys      se: Swedish (no dead keys)
  dvorak          se: Swedish (Dvorak)
  rus             se: Russian (Sweden, phonetic)
  rus_nodeadkeys  se: Russian (Sweden, phonetic, no dead keys)
  smi             se: Northern Saami (Sweden)
  mac             se: Swedish (Macintosh)
  svdvorak        se: Swedish (Svdvorak)
  us_dvorak       se: Swedish (based on US Intl. Dvorak)
  us              se: Swedish (US, with Swedish letters)
  swl             se: Swedish Sign Language
```
Checksums:
sha256sum:
e2707e232b5a7e112d2073350fed63ff5818f90bec95f28d09bb281470abe39c  rescuezilla-2.1.3-4-ga461f3bd-64bit.groovy.iso

sha1sum:
b37ffe2b9dac1e2d635f5ef9cef0bcfb0d763b6a  rescuezilla-2.1.3-4-ga461f3bd-64bit.groovy.iso

md5sum:
d9c14855b79fbfb1b80235e575641d11  rescuezilla-2.1.3-4-ga461f3bd-64bit.groovy.iso

Download: [rescuezilla-2.1.3-4-ga461f3bd-64bit.groovy.iso](https://drive.google.com/u/0/uc?id=1wrjYb8ENoExElyrHFsuCaTAKjPHfibh5&export=download)

**Comment by shasheene** [comment]
Translation now widely available as part of [Rescuezilla v2.2](https://github.com/rescuezilla/rescuezilla/releases/latest)
